### ISSUE TITLE
 How Install on Airtag?

### ISSUE THREAD

**Issue by michaelloliveira** [traccar/airtag-sync#1]
How to install the airtag-sync app on apple air tag?

**Comment by aljopro** [comment]
you don't install it on the AirTag. You have to have a MacOS system that can run Find My.

**Comment by michaelloliveira** [comment]
> you don't install it on the AirTag. You have to have a MacOS system that can run Find My.

How would I apply this to track remote objects?

**Comment by aljopro** [comment]
> How would I apply this to track remote objects?

Ok the idea is this. 

1. You setupthe AirTag  and put it on the thing you want to track. 
2. you run this script on your Mac that has the “Find My” app where it will query the app periodically for the coordinates of the AirTag.


This is only really something you would do if you have already bought into the apple ecosystem. If you haven’t bought any AirTags for like finding devices around the house then my suggestion would be to look elsewhere.

**Comment by michaelloliveira** [comment]
ok, but there is a limit of 32 tags.

Ideally, the tag would send location data to the traccar server.

**Comment by aljopro** [comment]
> ok, but there is a limit of 32 tags.
> 
> Ideally, the tag would send location data to the traccar server.

Agreed but Apple’s AirTag service a private API. Apple does not allow for outside applications to interface with AirTags.

this script takes advantage of a caching file that exists within the “find my” app. So 32 AirTags is what you get with this method.
